### ISSUE TITLE
age raster unavailable via url

### ISSUE THREAD

**Issue by louisRobert** [PredictiveEcology/SpaDES-modules#29]
Hello, new user here
 I do not know if this is an issue but the lcc2005 simulation fails for me with the error copied and pasted further below. I think it is a connectivity issue with an ftp website but since I am just trying the tutorial to learn about Spades, its very hard for me to pinpoint the exact error. 

In tutorial:  execute all line up to line

```r
mySim <- simInit(times = times, params = parameters, modules = modules,  paths = paths)
```

```r
###### Module Code Checking ########
Running .inputObjects for cropReprojectLccAge
Running preProcess
Preparing: can_age04_1km.tif
Checking local files...
Finished checking local files.
Can't download can_age04_1km.tif from ftp://ftp.daac.ornl.gov/data/nacp/NA_TreeAge//data/can_age04_1km.tif 
Please download and provide the object 'lcc05' as input for cropReprojectLccAge manually
```

**Comment by achubaty** [comment]
HI, this is a known problem caused by a change in the availability of that dataset. You will need to create an account and download the file manually.

http://daac.ornl.gov/data/nacp/NA_TreeAge//data/can_age04_1km.tif

**Comment by achubaty** [comment]
closed as duplicate of #12
